Rover/Rover.cpp

```diff
@@ -323,6 +323,10 @@ void Rover::ahrs_update()
     } else if (gps.status() >= AP_GPS::GPS_OK_FIX_3D) {
         ground_speed = ahrs.groundspeed();
     }
+    
+#if AP_FOLLOW_ENABLED
```

Comment:
Maybe put this just below the update?

Reply:
I think it's OK where it is actually.. it makes sense..